frontend/web/components/EditIdentity.tsx

```diff
@@ -73,48 +45,27 @@ const EditIdentity: FC<EditIdentityType> = ({ data, environmentId }) => {
     }
   }
 
-  const handleInput = () => {
-    if (aliasRef.current) {
-      const selection = window.getSelection()
-      const range = selection?.getRangeAt(0)
-      const cursorPosition = range?.startOffset || 0
-
-      const lowerCaseText = aliasRef.current.textContent?.toLowerCase() || ''
-      aliasRef.current.textContent = lowerCaseText
-
-      // Restore cursor position
-      const newRange = document.createRange()
-      newRange.setStart(
-        aliasRef.current.childNodes[0],
-        Math.min(cursorPosition, lowerCaseText.length),
-      )
-      newRange.collapse(true)
-
-      selection?.removeAllRanges()
-      selection?.addRange(newRange)
-    }
+  const handleInput = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const newValue = e.target.value.toLowerCase()
+    setAlias(newValue.replace(/\n/g, ' ').trim())
   }
 
   return (
     <>
-      <span
```

Comment:
I understood the span implementation was to deal with width. Switched to input for simplicity and easier maintainability (and compabilities cross-browser) while dealing with the width.
Let me know if there was another reason and I'll work on the compatibility